EIPS/eip-1.md

```diff
@@ -5,7 +5,7 @@ status: Active
 type: Meta
 author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, et al.
 created: 2015-10-27
-updated: 2015-12-07, 2016-02-01, 2018-03-21, 2018-05-29, 2018-10-17, 2019-05-19, 2019-12-04, 2020-06-17
+updated: 2015-12-07, 2016-02-01, 2018-03-21, 2018-05-29, 2018-10-17, 2019-05-19, 2019-12-04, 2020-06-17, 2020-09-10
```

Comment:
Isn't this information already in GitHub?  What is the purpose of an ever growing list of dates here?

Reply:
I agree here, it sure causes the front matter to look really cluttered.